saleor/core/db/connection.py

```diff
@@ -0,0 +1,76 @@
+import logging
+from typing import Any
+
+from django.conf import settings
+from django.core.management.color import color_style
+from django.db.backends.base.base import BaseDatabaseWrapper
+from django.db.backends.utils import CursorWrapper
+
+logger = logging.getLogger(__name__)
+
+
+def db_alias_logger_middleware(get_response):
+    def middleware(request):
+        from django.db import connections
+
+        for conn in connections.all():
+            wrap_connection(conn)
+
+        response = get_response(request)
+        return response
+
+    return middleware
+
+
+def wrap_connection(connection: BaseDatabaseWrapper):
+    if not hasattr(connection, "_orig_cursor"):
+        connection._orig_cursor = connection.cursor  # type: ignore
+
+        def cursor(*args, **kwargs):
+            orig_cursor = connection._orig_cursor(*args, **kwargs)  # type: ignore
+            PatchedCursor = _apply_mixin(orig_cursor.__class__, DbAliasLogger)
+            return PatchedCursor(orig_cursor.cursor, connection)
+
+        connection.cursor = cursor  # type: ignore[method-assign]
```

Comment:
I am not sure, but based on Django's code for `SimpleTestCase` class, we should also consider overriding these methods: `connect`, `temporary_connection,` and `chunked_cursor` but I could be wrong.